nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/tools/operations/AddWatermarkToVideo.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thibaud Arguillere
+ *     Ricardo Dias
+ */
+package org.nuxeo.ecm.platform.video.tools.operations;
+
+import org.apache.commons.lang.StringUtils;
+import org.nuxeo.ecm.automation.OperationException;
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.annotations.Param;
+import org.nuxeo.ecm.automation.core.collectors.BlobCollector;
+import org.nuxeo.ecm.automation.core.util.BlobList;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.DocumentModelList;
+import org.nuxeo.ecm.core.api.NuxeoException;
+import org.nuxeo.ecm.platform.video.tools.VideoToolsService;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * Watermark a Video with the given Picture, at the given position (from top-left).
+ *
+ * @since 8.4
+ */
+@Operation(id = AddWatermarkToVideo.ID, category = Constants.CAT_BLOB, label = "Watermarks a Video with a Picture", description = "Watermark the video with the picture stored in file:content of watermark, at the position(x, y) from the left-top corner of the picture.", aliases = {"Video.AddWatermark"})
```

Comment:
`aliases` are used when we change the operation ID so that it continues to work for people calling the old operation id. Here, there is no need to have an alias.